client/src/Pages/v1/StatusPage/Status/index.jsx

```
@@ -16,151 +16,204 @@ import { useIsAdmin } from "../../../../Hooks/v1/useIsAdmin.js";
+            <Typography variant="h2">{t("statusPageStatusServiceStatus")}</Typography>
+            <StatusBar monitors={monitors} />
+            <MonitorsList
+                monitors={monitors}
+                statusPage={statusPage}
+            />
+            
+            {link}
+
+            {/* --- CHANGE 4: Custom Footer Logic --- */}
+            {statusPage?.footerHTML && (
+                 <div dangerouslySetInnerHTML={{ __html: statusPage.footerHTML }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)